### PR TITLE
Made annotation-tools compatible to ASM 7 and deleting asmx

### PR DESCRIPTION
**Pull request by kushg1**
Deleted the entire asmx directory and made annotation-tools work with ASM version 7.2-beta, which is the latest release.

See https://github.com/typetools/checker-framework/pull/2796

Both pull requests need to be approved together.

### CODE REVIEW

File: annotation-file-utilities/tests/Makefile [hunk 1]
```diff
@@ -32,10 +32,14 @@ DEBUG :=
 # Use this to enable some debugging.
 # DEBUG := --debug
 
-default : all
+default : diffs
```
Comment (smillst) [inline annotation-file-utilities/tests/Makefile]: Why did the default change to `diffs`?  Why did you need to add the target `diffs`?

Comment (kushg1) [reply annotation-file-utilities/tests/Makefile]: Line 35 seems like a mistake on my part. I'll change this. 

I did add the `diffs` target to the makefile, and perhaps this should have been in a separate PR. I remember that this made it slightly easy for me to debug since I was using IntelliJ. This also seemed like the only way I would be able to run something like `make $(DIFFS)` from the command line.

**Comment by markro49** [comment]
This pull request was superseded by https://github.com/typetools/annotation-tools/pull/340.
